contracts/PartyBid.sol

```diff
@@ -183,7 +198,30 @@ contract PartyBid is Party {
             _ethFee = _closeSuccessfulParty(highestBid);
         }
         // set the contract status & emit result
-        emit Finalized(partyStatus, totalSpent, _ethFee, totalContributedToParty);
+        emit Finalized(partyStatus, totalSpent, _ethFee, totalContributedToParty, false);
+    }
+
+    // ======== External: Expire =========
+
+    /**
+     * @notice Expires an auction, moving it to LOST state and ending the ability to contribute.
+     * @dev Emits a Finalized event upon success; callable by anyone once the expiration date passes.
+     */
+    function expire() external nonReentrant {
+        require(
+            partyStatus == PartyStatus.ACTIVE,
+            "PartyBid::expire: auction not active"
+        );
+        require(
+            address(this) !=
+                marketWrapper.getCurrentHighestBidder(
+                    auctionId
+                ),
+            "PartyBid::expire: currently highest bidder"
+        );
```

Comment:
should we require that `address(this).balance >= totalContributedToParty` ? theoretically that should be the case if the `highestBidder` check passes, but given different folks implement various auction standards, it could be valuable to have redundancy

Reply:
The idea being that if we have an active bid we must have sent in the eth to the auction, and this should be the only reason the balance isn't equal to the total contributed?  I think that sounds fine